elasticache-redis/main.tf

```diff
@@ -132,6 +102,17 @@ resource "aws_ssm_parameter" "endpoint" {
   key_id = aws_kms_key.ssm_redis_endpoint.arn
 }
 
+resource "aws_ssm_parameter" "redis_url" {
+  name        = "/copilot/${var.application}/${var.environment}/secrets/${upper(replace("${var.name}_URL", "-", "_"))}"
+  description = "Redis endpoint with SSL configuration"
+  type        = "SecureString"
+  value       = "rediss://${aws_elasticache_replication_group.redis.primary_endpoint_address}:6379?ssl_cert_reqs=CERT_REQUIRED"
+
+  tags = local.tags
+
+  key_id = aws_kms_key.ssm_redis_endpoint.arn
```

Comment:
This would be only be known after an apply I believe. But there should be a comment in the unit tests to point at the e2e tests
![Screenshot 2024-05-31 at 15 25 08](https://github.com/uktrade/terraform-platform-modules/assets/54268925/08cd0dd5-376e-4856-aaae-f3adc1d4d828)

Reply:
We could follow this pattern: https://github.com/uktrade/terraform-platform-modules/blob/main/environment-pipelines/tests/unit.tftest.hcl#L197